programmers/kim.js

```diff
@@ -0,0 +1,15 @@
+function solution(seoul){
+    var location = seoul.indexOf("Kim");
+    var location2 = seoul.lastIndexOf("Kim");
+    if( location === location2){
+        return "김서방은 " + location + "에 있다."
+    }
+    else if (location === location2=== -1){
+        console.log("김서방은 최소 한 곳에는 있어야 한다.")
+    }
+    else {
+        console.log("김서방은 한 곳에만 있을 수 있다.");
+    } 
+}
```

Comment:
코딩테스트에서는 주어진 것에 대한 것만 풀이해주시면 됩니다.
else if를 쓰실 때에는 } else if ()  중괄호 옆에 써주시는 것이 보기 좋습니다.
```suggestion
    if (location === location2) {
        return `김서방은 ${location}에 있다`;
    } 
```